tests/test_chiral_utils.py

```diff
@@ -138,3 +139,160 @@ def test_find_chiral_bonds():
+
+  5  4  0  0  0  0  0  0  0  0999 V2000
+    0.0051   -0.0106    0.0060 C   0  0  0  0  0  0  0  0  0  0  0  0
+    0.5497    0.7554   -0.5970 H   0  0  0  0  0  0  0  0  0  0  0  0
+    0.7498   -0.5879    0.5853 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.5868   -0.6521   -0.6761 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.7178    0.4953    0.6818 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0
+  1  3  1  0
+  1  4  1  0
+  1  5  1  0
+M  END"""
+
+molblock_N = """
+     RDKit          3D
+
+  4  3  0  0  0  0  0  0  0  0999 V2000
+    0.0195   -0.0020    0.2429 N   0  0  0  0  0  0  0  0  0  0  0  0
+    0.9942   -0.1240   -0.0852 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.5944   -0.7730   -0.0788 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.4193    0.8989   -0.0789 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0
+  1  3  1  0
+  1  4  1  0
+M  END"""
+
+molblock_CC = """
+     RDKit          3D
+
+  8  7  0  0  0  0  0  0  0  0999 V2000
+   -0.7455    0.0414    0.0117 C   0  0  0  0  0  0  0  0  0  0  0  0
+    0.7473    0.0029    0.0012 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.1297   -0.6374    0.8144 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.1849    1.0256    0.1996 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.1999   -0.3346   -0.9389 H   0  0  0  0  0  0  0  0  0  0  0  0
+    1.0842   -0.7365   -0.7732 H   0  0  0  0  0  0  0  0  0  0  0  0
+    1.2266    0.9617   -0.2681 H   0  0  0  0  0  0  0  0  0  0  0  0
+    1.2019   -0.3231    0.9532 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0
+  1  3  1  0
+  1  4  1  0
+  1  5  1  0
+  2  6  1  0
+  2  7  1  0
+  2  8  1  0
+M  END"""
+
+molblock_CN = """
+     RDKit          3D
+
+  7  6  0  0  0  0  0  0  0  0999 V2000
+   -0.5732    0.0243   -0.0031 C   0  0  0  0  0  0  0  0  0  0  0  0
+    0.8233   -0.0403   -0.0341 N   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.0026   -0.4265    0.9182 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.9088    1.0746   -0.0741 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.9930   -0.5345   -0.8755 H   0  0  0  0  0  0  0  0  0  0  0  0
+    1.2903   -0.9052    0.2005 H   0  0  0  0  0  0  0  0  0  0  0  0
+    1.3640    0.8076   -0.1320 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0
+  1  3  1  0
+  1  4  1  0
+  1  5  1  0
+  2  6  1  0
+  2  7  1  0
+M  END"""
+
+
+def test_chiral_conflict_flip():
+    # exercise case of no conflicts or a flip conflict
+
+    mol_a = Chem.MolFromMolBlock(molblock_C, removeHs=False)
+    mol_b = Chem.MolFromMolBlock(molblock_C, removeHs=False)
+
+    conf_a = mol_a.GetConformer(0).GetPositions()
+    conf_b = mol_b.GetConformer(0).GetPositions()
+
+    chiral_set_a = ChiralRestrIdxSet.from_mol(mol_a, conf_a)
+    chiral_set_b = ChiralRestrIdxSet.from_mol(mol_b, conf_b)
+
+    assert len(chiral_set_a.restr_idxs) == 4
+    assert len(chiral_set_b.restr_idxs) == 4
+
+    identity_map = np.array([(i, i) for i in range(len(conf_a))])
+
+    swap_map = np.array(identity_map)
```

Comment:
nit: Add comment # swap hydrogens to flip chirality

Reply:
Added in https://github.com/proteneer/timemachine/pull/908/commits/48f909e92fef38757cfc002b3ff10ca62e131a0f